core/solver/batch_bicgstab.cpp

```diff
@@ -56,8 +56,7 @@ std::unique_ptr<BatchLinOp> BatchBicgstab<ValueType>::transpose() const
     return build()
         .with_preconditioner(parameters_.preconditioner)
         .with_max_iterations(parameters_.max_iterations)
-        .with_rel_residual_tol(parameters_.rel_residual_tol)
-        .with_abs_residual_tol(parameters_.abs_residual_tol)
+        .with_residual_tol(parameters_.residual_tol)
```

Comment:
Is it possible to take separate tolerances as before here, but have only one tolerance in the kernel based on the tolerance type? As a user interface, it makes sense to take two separate tolerances because we might need to use them both in future. Within the kernel, we prioritize efficiency for now and keep only one, but we could change that any time without breaking interface.

Reply:
I have changed it back now.